test_task_packed_picking/__manifest__.py

```diff
@@ -7,7 +7,7 @@
     "category": "Test",
     "license": "LGPL-3",
     "depends": [
-        "base",
+        "sale_management",
```

Comment:
Why do we need sale_management here?
If you really need both sale and stock  there is a [sale_stock](https://github.com/odoo/odoo/tree/16.0/addons/sale_stock) module which you can inherit